sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/api/Azure.ResourceManager.SpringAppDiscovery.netstandard2.0.cs

```diff
@@ -0,0 +1,414 @@
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public SpringAppDiscoveryProvisioningState(string value) { throw null; }
+        public static Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState Accepted { get { throw null; } }
+        public static Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState Canceled { get { throw null; } }
+        public static Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState Deleting { get { throw null; } }
+        public static Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState Failed { get { throw null; } }
+        public static Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState Provisioning { get { throw null; } }
+        public static Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState Succeeded { get { throw null; } }
+        public static Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState Unknown { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState left, Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState right) { throw null; }
+        public static implicit operator Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState left, Azure.ResourceManager.SpringAppDiscovery.Models.SpringAppDiscoveryProvisioningState right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class SpringBootAppApplicationConfigurationsItem
+    {
+        public SpringBootAppApplicationConfigurationsItem(string key) { }
+        public string Key { get { throw null; } set { } }
+        public string Value { get { throw null; } set { } }
+    }
+    public partial class SpringBootAppInstancesItem
+    {
+        public SpringBootAppInstancesItem(string machineArmId) { }
+        public int? InstanceCount { get { throw null; } set { } }
+        public int? JvmMemoryInMB { get { throw null; } set { } }
+        public string MachineArmId { get { throw null; } set { } }
```

Comment:
the name indicates this is an arm resource id, is it?
If so, we should change the format of this property to `arm-id`
You could do it by adding this into `rename-mapping` :
```
rename-mapping:
   <key>: -|arm-id
```
please find the <key> with mgmt-debug on from the comment of this property.